From b4d121d84b4ade190126d6e1ca01bfd1cda7eece Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Wed, 30 Nov 2022 02:00:35 +0000
Subject: [PATCH] src/frontend/qt/qtkey.h: add missing <stdint.h> import (#520)

Without the change build fails on weekly `gcc-13` snapshot as:

    frontend/qt/qtkey.h:25:19: error: 'uint32_t' was not declared in this scope
       25 | int keysymToQtKey(uint32_t keysym, const QString &text);
          |                   ^~~~~~~~
---
 src/frontend/qt/qtkey.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/frontend/qt/qtkey.h b/src/frontend/qt/qtkey.h
index 4011ff56..2279cce0 100644
--- a/src/frontend/qt/qtkey.h
+++ b/src/frontend/qt/qtkey.h
@@ -20,6 +20,7 @@
 #ifndef _PLATFORMINPUTCONTEXT_QTKEY_H_
 #define _PLATFORMINPUTCONTEXT_QTKEY_H_
 
+#include <stdint.h>
 #include <QString>
 
 int keysymToQtKey(uint32_t keysym, const QString &text);
-- 
2.40.1

